src/components/MainBody.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import TodoItem from './TodoItem'
+
+function MainBody(props) {
+    return (
+        <section className="main" style={{display: 'block'}}>
+            <input id="toggle-all" className="toggle-all" type="checkbox"/>
+            <label htmlFor="toggle-all">Mark all as complete</label>
+            <ul className="todo-list">
+                {props.todoItems.length ?
+                    props.todoItems.map((todoItem, number) => {

```

Comment:
Using `map` on empty array returns an empty array. It means nothing will be rendered. Checking `length` here is redundant. 
```jsx
            <ul className="todo-list">
                {props.todoItems.map((todoItem, number) => {
```